graphql/user/fields.ts

```diff
@@ -55,6 +57,54 @@ export class Contact {
     @Field((_type) => String, { nullable: true })
     chatId?: string;
 }
+
+@ObjectType()
+export class ImportantInformationNew {
+    @Field((_type) => String)
+    title: string;
+    @Field((_type) => String)
+    description: string;
+    @Field((_type) => String)
+    type: 'normal' | 'sequence';
+
+    @Field((_type) => Int, { nullable: true })
+    maxSteps: number | null;
+    @Field((_type) => Int, { nullable: true })
+    finishedSteps: number | null;
+    // @Field((_type) => important_information_recipients_enum)
+    @Field((_type) => String)
+    recipients: 'students' | 'pupils';
+    // @Field((_type) => important_information_language_enum)
+    @Field((_type) => String)
+    language: 'en' | 'de';
+    @Field((_type) => ImportantInformationModal)
+    modal: typeof ImportantInformationModal;
```

Comment:
To be able to show the achievement modal, we have to have all the required information. Therefore, I would introduced a modal union.

Reply:
Can't we return the achievements through the regular achievement resolver? And only the important information through a different one? I would prefer to keep the GraphQL interface simple (and hide all the complexity of the achievement system behind it)

Reply:
(downside is that we potentially evaluate twice, but we could potentially cache in the User Context)

Reply:
So you mean that we should remove the Achievement from the ImportantInformationModal and only provide an achievement ID instead? Wouldn't this just move the complexity from the graphql API to the Frontend?

Reply:
I meant

```
me {
  achievements {
    // achievement data
  }

  importantInformations {
    // important information
  }
}
```

Reply:
To avoid the union and keep the concepts separate at the API layer

Reply:
But how do we show the achievements within the important information then? The same way typedigital had in mind back then?

https://github.com/corona-school/user-app/pull/546

Reply:
Yes, the UI is different for the two anyways? So dont see that much difference between looping a union and looping two different arrays. 

There might be other places where we want to show just the achievements but not the important information (vice-versa)